frontend/homework/5/index.js

```diff
@@ -0,0 +1,46 @@
+const http = require('http');
+const os = require('os');
+const fs = require('fs');
+
+function getSystemInfo() {
+    return {
+        HostName: os.hostname(),
+        OperatingSystem: os.platform(),
+        Architecture: os.arch(),
+        OSRelease: os.release(),
+        Uptime: os.uptime(),
+        NumberOfCPUCores: os.cpus().length,
+        TotalMemory: os.totalmem(),
+        FreeMemory: os.freemem(),
+        CurrentWorkingDirectory: process.cwd()
+    };
+}
+
+function writeSystemInfoToFile() {
+    const systemInfo = getSystemInfo();
+    const jsonContent = JSON.stringify(systemInfo, null, 2);
+    fs.writeFileSync('system_info.json', jsonContent);
+}
+
+writeSystemInfoToFile();
+
+const server = http.createServer((req, res) => {
+    fs.readFile('system_info.json', (err, data) => {
```

Comment:
Needed to check the API format and according to that further process must be done, also the response format must be matched with the provided format in the doc